_data/roles.yml

```diff
@@ -52,6 +66,11 @@
     Preparing a short report about all general developments in OBO Technical Working Group (TWG)
      for the OBO Operations team meeting. You will go through pull requests and emails and summarize 
      in bullet points and report to OBO Ops.
+  lead(s):
+      - name: Ray Stefancsik
```

Comment:
ORCID and github plz

Reply:
Please don't add any new people yet. There will be a public process with official invites etc. For now list me everywhere and I will manage the transition with an official process

Reply:
I added the github and orcids

@bvarner-ebi and @shawntanzk could you please add your orcid?

Reply:
I would say don't spend time on metadata collection past ORCID since we can automate grabbing any other relevant metadata, but here they are:

- @shawntanzk https://orcid.org/0000-0001-7258-9596
- @zhengj2007 https://orcid.org/0000-0002-2999-0103
- @bvarner-ebi https://orcid.org/0000-0002-1773-2692